variants/wio-e5/platformio.ini

```diff
@@ -1,39 +1,27 @@
 [env:wio-e5]
 extends = stm32_base
 board = lora_e5_dev_board
-board_upload.maximum_size = 233472 ; reserve the last 28KB for filesystem
+board_upload.maximum_size = 233472 ; reserve the last 128KB for filesystem
```

Comment:
This is actually only 28kb ( (262144-233472) / 1024) = 28kb

Reply:
yes, the 28kb is the correct value. the nRF52840 uses the same flash size.